package.json

```diff
@@ -31,5 +31,6 @@
             "_comment": "This will remove husky from when we started migrating to use prettier",
             "pre-commit": "npm uninstall husky"
         }
-    }
+    },
+    "dependencies": {}
```

Comment:
Please revert these changes to the root `package.json`, thanks!

Reply:
Apologies missed this!